<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GBSNaviAI</class>
 <widget class="QWidget" name="GBSNaviAI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>205</width>
    <height>920</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QGroupBox" name="verticalNavi">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>205</width>
     <height>919</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="imgTheme">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>720</y>
      <width>101</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>720</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="imgAvator">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>780</y>
      <width>48</width>
      <height>48</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblHelloMessage">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>780</y>
      <width>76</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblNickName">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>800</y>
      <width>76</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblNaviTitle">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>65</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="spliteLine">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>205</width>
      <height>2</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:green;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>40</y>
      <width>211</width>
      <height>681</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout"/>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
